### DISCUSSION TITLE
Limit JSON output (e.g. last 5 events)

### DISCUSSION THREAD

**Discussion by nubbeldupp** [br-g/openf1#67] [Q&A] [answered]
Hi,

I found this API and it's awesome. 
I'm using it to add Widgets to my Homelab-Website using [gethomepage/homepage](https://github.com/gethomepage/homepage).
Unfortunately I can't go through the array. I have to specify the data I want. 
<img width="1000" alt="Screenshot 2024-05-05 at 02 11 23" src="https://github.com/br-g/openf1/assets/2195212/9cc77ae5-8fea-4064-b18d-2e973afdc156">
<img width="606" alt="Screenshot 2024-05-05 at 02 11 16" src="https://github.com/br-g/openf1/assets/2195212/910fbc69-8663-4d9b-af03-e039de71d632">

Is there a way to limit the Output to the last e.g. 5 events using a filter in the URL? 
For example something like this:

api.openf1.org/v1/pit?session_key=latest&**_fetch_last=5_**

Kind regards

**Comment by nubbeldupp** [comment]
I found a sort_by in the db.py but I'm not sure if I can or  how to use it. 
If I can sort the data DESC/-1 by _id or the date I could just grab the objects from the array I want/need.

**Comment by br-g** [comment, marked as answer]
Sorry this feature does not yet exist.
It has been requested a lot though, so I am seriously considering it!
